_about/notes/0.5.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Istio 0.5
+
+order: 96
+
+layout: about
+type: markdown
+---
+{% include home.html %}
+
+In addition to the usual pile of bug fixes and perf improvements, this release includes the new or 
+updated features detailed below.
+
+## Networking
+
+- **Control Plane Deployments**. `istioctl` now has a deployment command to produce istio control plane deployments. TODO: Doc link?
+
+- **Automatic Proxy Injection**. Use Kubernetes 1.9's mutating webhook functionality to provide automatic
```

Comment:
Webhook based proxy injection

Reply:
Do users care about the mechanism (webhook vs. initializer)? Manual vs. automatic is used elsewhere in the documentation. Alternatively, we could use client-side vs. server-side.